lib/PreconditionRunner.pm

```diff
@@ -95,12 +96,53 @@ sub write_num_passes
 sub run_to_steady_state($)
 {
     my $self = shift;
-    
     my %args = @_;
 
     my $msg_prefix = $args{'msg_prefix'};
-    my $output_dir = $args{'output_dir'};
     my $test_ref = $args{'test_ref'};
+    my $output_dir = $args{'output_dir'};
+
+    # preconditioner.exe does not support multiple files
```

Comment:
Interesting.... I think I understand what it means to drive a *device* to steady-state, but I'm not sure what it means to drive more than one file on a single device to steady-state.  Hmm...